tools/classes.js

```diff
@@ -150,7 +150,9 @@ var classes = [
   'foam.nanos.script.Language',
   'foam.nanos.auth.Language',
   'foam.nanos.auth.Region',
+  'foam.nanos.logger.DiscardLogger',
```

Comment:
Rename DiscardLogger here to LogLevelFilterLogger.

Reply:
Did that, added RepeatLogger as well. 

The services file needs to be updated as well, its on my radar.